examples/lb/listener/main.tf

```diff
@@ -0,0 +1,6 @@
+resource "vkcs_lb_listener" "listener" {
+  name            = "listener-app-http"
```

Comment:
do not forget to add tf-example to names

Reply:
> tf-example

added everywhee where field name is

Reply:
> and add descriptions where it is possible

added everywhere possible